docs/conf.py

```diff
@@ -56,9 +56,9 @@ def setup(sphinx):
 # built documents.
 #
 # The short X.Y version.
-version = '0.2.0'
+version = '0.4.1'
```

Comment:
I crated #1088 for this.

@nicksavers if you are already working on this, let others know by assigning #1088 to yourself.